### ISSUE TITLE
ppyolov2训练一轮后出现暂停

### ISSUE THREAD

**Issue by ANewAnonymous** [PaddlePaddle/PaddleDetection#2857]
我使用静态图中config/ppyolo/ppyolov2_r101vd_dcn.yml配置文件训练自定义的VOC类型数据集，在完成第一轮训练后，评估时卡住了，无任何提示，程序仍然在继续运行，但没有进行训练。
如下情况:
`[INFO 2021-05-03 13:19:13,491 train.py:299] iter: 1000, lr: 0.000250, 'loss_xy': '11.308865', 'loss_wh': '28.035952', 'loss_obj': '230.639816', 'loss_cls': '7.170611', 'loss_iou': '43.606972', 'loss_iou_aware': '8.102552', 'loss': '308.150330', eta: 1 day, 10:12:40, batch_cost: 0.82658 sec, ips: 14.51770 images/sec
2021-05-03 13:19:14,420 - INFO - Save model to output/ppyolov2/1000.
[INFO 2021-05-03 13:19:14,420 checkpoint.py:199] Save model to output/ppyolov2/1000.
2021-05-03 13:19:51,682 - INFO - Test iter 0
[INFO 2021-05-03 13:19:51,682 eval_utils.py:189] Test iter 0`

配置文件如下:
`architecture: YOLOv3
use_gpu: true
max_iters: 150000
log_iter: 20
save_dir: output
snapshot_iter: 1000
metric: VOC
pretrain_weights: https://paddle-imagenet-models-name.bj.bcebos.com/ResNet101_vd_ssld_pretrained.tar
weights: output/ppyolov2_r101vd_dcn/model_final
num_classes: 1
use_fine_grained_loss: true
use_ema: true
ema_decay: 0.9998

YOLOv3:
  backbone: ResNet
  yolo_head: YOLOv3PANHead
  use_fine_grained_loss: true

ResNet:
  norm_type: sync_bn
  freeze_at: 0
  freeze_norm: false
  norm_decay: 0.
  depth: 101
  feature_maps: [3, 4, 5]
  variant: d
  dcn_v2_stages: [5]

YOLOv3PANHead:
  anchor_masks: [[6, 7, 8], [3, 4, 5], [0, 1, 2]]
  anchors: [[10, 13], [16, 30], [33, 23],
            [30, 61], [62, 45], [59, 119],
            [116, 90], [156, 198], [373, 326]]
  norm_decay: 0.
  iou_aware: true
  iou_aware_factor: 0.5
  scale_x_y: 1.05
  spp: true
  yolo_loss: YOLOv3Loss
  nms: MatrixNMS
  drop_block: true

YOLOv3Loss:
  ignore_thresh: 0.7
  scale_x_y: 1.05
  label_smooth: false
  use_fine_grained_loss: true
  iou_loss: IouLoss
  iou_aware_loss: IouAwareLoss

IouLoss:
  loss_weight: 2.5
  max_height: 768
  max_width: 768

IouAwareLoss:
  loss_weight: 1.0
  max_height: 768
  max_width: 768

MatrixNMS:
  background_label: -1
  keep_top_k: 100
  normalized: false
  score_threshold: 0.01
  post_threshold: 0.01

LearningRate:
  base_lr: 0.001
  schedulers:
  - !PiecewiseDecay
    gamma: 0.1
    milestones:
    - 120000
    - 135000
  - !LinearWarmup
    start_factor: 0.
    steps: 4000

OptimizerBuilder:
  clip_grad_by_norm: 35.
  optimizer:
    momentum: 0.9
    type: Momentum
  regularizer:
    factor: 0.0005
    type: L2

TrainReader:
  inputs_def:
    fields: ['image', 'gt_bbox', 'gt_class', 'gt_score']
    num_max_boxes: 100
  dataset:
    !VOCDataSet
      dataset_dir: /home/aistudio/work/dataset
      anno_path: lable_train/train_list.txt
      label_list: /home/aistudio/work/dataset/lable_train/labels.txt
      with_background: false
      use_default_label: false
  sample_transforms:
    - !DecodeImage
      to_rgb: True
      with_mixup: True
    - !MixupImage
      alpha: 1.5
      beta: 1.5
    - !ColorDistort {}
    - !RandomExpand
      ratio: 2.0
      fill_value: [123.675, 116.28, 103.53]
    - !RandomCrop {}
    - !RandomFlipImage
      is_normalized: false
    - !NormalizeBox {}
    - !PadBox
      num_max_boxes: 100
    - !BboxXYXY2XYWH {}
  batch_transforms:
  - !RandomShape
    sizes: [320, 352, 384, 416, 448, 480, 512, 544, 576, 608, 640, 672, 704, 736, 768]
    random_inter: True
  - !NormalizeImage
    mean: [0.485, 0.456, 0.406]
    std: [0.229, 0.224, 0.225]
    is_scale: True
    is_channel_first: false
  - !Permute
    to_bgr: false
    channel_first: True
  - !Gt2YoloTarget
    anchor_masks: [[6, 7, 8], [3, 4, 5], [0, 1, 2]]
    anchors: [[10, 13], [16, 30], [33, 23],
              [30, 61], [62, 45], [59, 119],
              [116, 90], [156, 198], [373, 326]]
    downsample_ratios: [32, 16, 8]
  batch_size: 12
  shuffle: true
  mixup_epoch: 25000
  drop_last: true
  worker_num: 8
  bufsize: 4
  use_process: true

EvalReader:
  inputs_def:
    fields: ['image', 'im_size', 'im_id']
    num_max_boxes: 100
  dataset:
    !VOCDataSet
      dataset_dir: /home/aistudio/work/dataset
      anno_path: lable_train/test_list.txt
      label_list: /home/aistudio/work/dataset/lable_train/labels.txt
      with_background: false
      use_default_label: false
  sample_transforms:
    - !DecodeImage
      to_rgb: True
    - !ResizeImage
      target_size: 640
      interp: 2
    - !NormalizeImage
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]
      is_scale: True
      is_channel_first: false
    - !PadBox
      num_max_boxes: 50
    - !Permute
      to_bgr: false
      channel_first: True
  batch_size: 8
  drop_empty: false
  worker_num: 8
  bufsize: 4

TestReader:
  inputs_def:
    image_shape: [3, 640, 640]
    fields: ['image', 'im_size', 'im_id']
  dataset:
    !ImageFolder
      anno_path: /home/aistudio/work/dataset/lable_train/labels.txt
      with_background: false
      use_default_label: false
  sample_transforms:
    - !DecodeImage
      to_rgb: True
    - !ResizeImage
      target_size: 640
      interp: 2
    - !NormalizeImage
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]
      is_scale: True
      is_channel_first: false
    - !Permute
      to_bgr: false
      channel_first: True
  batch_size: 1
`

训练使用的命令:
`python tools/train.py -c ~/work/ppyolov2.yml --eval --vdl_log_dir ~/log/ppyolov2  --use_vdl True`

**Comment by jerrywgz** [comment]
可以先使用top -c看下cpu占用率，如果卡住的时候占用率较高，可以尝试将trainreader中的use_process设置为false
